https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/merge_requests/37
(see also https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/merge_requests/38)

From de1b9c444df1ed66d72a4ab3d0e4dd2151037934 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Thu, 14 Sep 2023 16:12:32 +0200
Subject: [PATCH] file_wrapper.h: add missing include for musl

this fixes:
| In file included from ../webrtc-audio-processing-1.3/webrtc/rtc_base/system/file_wrapper.cc:11:
| ../webrtc-audio-processing-1.3/webrtc/rtc_base/system/file_wrapper.h:86:21: error: 'int64_t' has not been declared

if built with musl libc

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 webrtc/rtc_base/system/file_wrapper.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/webrtc/rtc_base/system/file_wrapper.h b/webrtc/rtc_base/system/file_wrapper.h
index 42c463c..c34d366 100644
--- a/webrtc/rtc_base/system/file_wrapper.h
+++ b/webrtc/rtc_base/system/file_wrapper.h
@@ -13,6 +13,7 @@
 
 #include <stddef.h>
 #include <stdio.h>
+#include <cstdint>
 
 #include <string>
 
-- 
GitLab
